homeassistant/components/blink/binary_sensor.py

```diff
@@ -48,8 +56,20 @@ def __init__(self, data, camera, description: BinarySensorEntityDescription):
         self.entity_description = description
         self._attr_name = f"{DOMAIN} {camera} {description.name}"
         self._camera = data.cameras[camera]
+        self._camera_name = camera
         self._attr_unique_id = f"{self._camera.serial}-{description.key}"
 
+    @property
+    def device_info(self):
```

Comment:
As above, please use dataclass and attr format: `self._attr_device_info = DeviceInfo(...)`